services/main-frontend/Dockerfile.production.cache.dockerfile

```diff
@@ -18,5 +18,6 @@ RUN npm run postinstall
 
 ENV NEXT_PUBLIC_BASE_PATH=""
 ENV NEXT_PUBLIC_SITE_TITLE="MOOC.fi courses"
+ENV NEXT_PUBLIC_PUBLIC_ADDRESS="https://courses.mooc.fi"
```

Comment:
Use kubernetes configs and this: https://nextjs.org/docs/api-reference/next.config.js/runtime-configuration